xbmc/video/VideoDatabase.cpp

```diff
@@ -3444,32 +3441,6 @@ void CVideoDatabase::GetDetailsFromDB(const dbiplus::sql_record* const record, i
 DWORD movieTime = 0;
 DWORD castTime = 0;
 
-CVideoInfoTag CVideoDatabase::GetDetailsByTypeAndId(VIDEODB_CONTENT_TYPE type, int id)
```

Comment:
Please split this into a separate commit as it's not related to the optimization.